internal/service/advancedcluster/data_source_advanced_clusters.go

```diff
@@ -14,7 +14,7 @@ import (
 	matlas "go.mongodb.org/atlas/mongodbatlas"
 )
 
-func DataSourceAdvancedClusters() *schema.Resource {
+func PluralDataSource() *schema.Resource {
```

Comment:
(not blocking) Interesting name 🤔 What do you think about `DataSources` instead? 
```suggestion
func DataSources() *schema.Resource {
```

Reply:
I'm following what @AgustinBettati started, not a strong opinion to either